### ISSUE TITLE
Compilation and Libraries issue

### ISSUE THREAD

**Issue by SridharanMurugadass** [aadhitya98/iplteamspdf#1]
there is only one issue in compilation there is one extra bracket found while closing the class ipl.java and then the swing project was properly working, due to the configuration missed in the form.

And please try to use apache poi instead of itext for building pdf files. Since its from apache foundation performance as well as the look and feel will be good.

other than that it was good @aadhitya98

**Comment by aadhitya98** [comment]
Thank u sir... I will learn how to use apache... Is the project running
successfully sir?

**Comment by SridharanMurugadass** [comment]
yes @aadhitya98 its working good.... great job.... keep it up.... please don't hesitate to connect with for any help.... thanks....

**Comment by aadhitya98** [comment]
Thank u sir... will definetly connect with u ....
